pkg/util/exec-run.go

```diff
@@ -68,6 +70,28 @@ func (r RealRunner) RunStdoutPipe(command string, args ...string) ([]byte, error
 	return data, nil
 }
 
+// RunCommandWithLog triggers command passed as arguments and it also does the
+// following things before command completion
+// 1. Logs the stdout of command to stdout(standard output)
+// 2. Logs stderr of the command to standard error
+func (r RealRunner) RunCommandWithLog(command string, args ...string) ([]byte, error) {
+	// #nosec
+	cmd := exec.Command(command, args...)
+	// Redirect the command output to stdout
+	cmd.Stdout = os.Stdout
+	// Redirect the command output to stderr
+	cmd.Stderr = os.Stderr
+	// Start the command
+	if err := cmd.Start(); err != nil {
```

Comment:
We can think of Run() command in later PR that may help to avoid few lines of code. But it is good for now.